### ISSUE TITLE
Question about usage

### ISSUE THREAD

**Issue by amilajack** [gajus/eslint-plugin-flowtype#67]
I am having issues with this plugin not showing me flow errors. I have all the rules at the strictest config:

```
    "flowtype/define-flow-type": 2,
    "flowtype/require-parameter-type": 2,
    "flowtype/require-return-type": [
      2,
      "always",
      {
        "annotateUndefined": "never"
      }
    ],
    "flowtype/space-after-type-colon": [
      2,
      "always"
    ],
    "flowtype/space-before-type-colon": [
      2,
      "never"
    ],
    "flowtype/type-id-match": [
      2,
      "^([A-Z][a-z0-9]+)+Type$"
    ],
    "flowtype/use-flow-type": 2,
    "flowtype/valid-syntax": 2
```

and I have the following code:

This will return the error: 'Missing "player" type annotation'
```js
setPlayer(player) {
  this.setState({ currentPlayer: player });
}
```

While this will not show any errors
```js
export function square(n: number): number {
  return n + { foo: 'loo' };
}
```

**Comment by danharper** [comment]
This won't actually run Flow. It only enforces linting rules for the Flow syntax.

So in your first example, "Missing "player" type annotation" is the lint error for `flowtype/require-parameter-type`. The second example passes lint, but you need to run `flow` for the error it'll give.

**Comment by amilajack** [comment]
I see. Is there a way to get flow errors in eslint?

**Comment by danharper** [comment]
Not that I'm aware, no. Personally I'd recommend an editor which supports ESLint & Flow and can show errors inline. (e.g. Nuclide or VS Code with the Flow plugin)

**Comment by amilajack** [comment]
Do you think this could be possible?
